_internal/config.yaml

```diff
@@ -14,8 +14,19 @@ enabled_previews:
   - audio
   - markdown
 
+## Do we preview the README if one is present
+enabled_readme: true
```

Comment:
readme_enabled?

Reply:
If it's false the README isn't displayed, even if there is one

Reply:
Ye I meant `readme_enabled` as key maybe

Reply:
it was to follow the syntax of the one before

Reply:
Ye the one above is the enabled previews, this one is whether the readme is enabled so `readme_enabled` probably sounds better but that's a nit.

Reply:
But I feel like having a line enabled_* and the next one *_enabled would be confusing